wp-content/themes/reconasia/assets/_scss/pages/archive.scss

```diff
@@ -1,3 +1,68 @@
 @use '../abstracts' as *;
 
 @use '../components/entry-header';
+
+.pagination {
+  &,
+  &__results {
+    @extend %text-style-label-medium;
+  }
+
+  &__results {
+    margin-top: rem(32);
+    margin-bottom: rem(32);
+    padding-bottom: rem(8);
+    color: $color-black-190;
+    border-bottom: 1px solid $color-border-dark-130;
+  }
+
+  .nav-links {
+    display: flex;
+    gap: rem(32);
```

Comment:
This needs to be less on mobile.  If you go to page 4 of the [results](https://localhost:3000/page/4/?s=Belt), you can see the max width of the content with ellipsis and all of the things.

Reply:
Good call. I ended up reducing the gap to 20px on the smallest screen size as 24px was still too wide for this circumstance:

<img width="404" alt="Screen Shot 2021-03-24 at 5 00 32 PM" src="https://user-images.githubusercontent.com/1896496/112382708-94079100-8cc2-11eb-900f-d7c9e27814e7.png">